### ISSUE TITLE
`before_delete_page` hook documentation clarification needed

### ISSUE THREAD

**Issue by KalobTaulien** [wagtail/wagtail#6499]
On https://docs.wagtail.io/en/stable/reference/hooks.html#before-delete-page it says:

> Called at the beginning of the “delete page” view passing in the request and the page object.
> Uses the same behaviour as before_create_page.

That's referring to [this area](https://docs.wagtail.io/en/stable/reference/hooks.html#before-create-page) and the following code:
```python
from wagtail.core import hooks

from .models import AwesomePage
from .admin_views import edit_awesome_page

@hooks.register('before_create_page')
def before_create_page(request, parent_page, page_class):
    # Use a custom create view for the AwesomePage model
    if page_class == AwesomePage:
        return create_awesome_page(request, parent_page)
```

However, according to https://github.com/wagtail/wagtail/blob/master/wagtail/admin/views/pages/delete.py#L20 the `before_delete_page ` hook only takes `request` and `page` as parameters.

I suggest the documentation around the `before_delete_page` hook be updated, ideally with a code example (even if it's somewhat duplicating another similar example). 

This is a good first issue for anybody to pick up.

**Comment by Tanmay-47** [comment]
Hey, If this issue is still open, I'm willing to take this up.

**Comment by lb-** [comment]
fixed via 92381698d0215d9f26c71a52c61ddeb429fb96e7  & https://github.com/wagtail/wagtail/pull/6500
